library/build.gradle

```diff
@@ -69,9 +70,9 @@ uploadArchives {
     repositories.mavenDeployer {
         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-        repository(url: sonatypeRepo) {
```

Comment:
`sonatypeRepo` is not found when building. If this is in a properties file that is gitignore, maybe add a check so other users can compile their forks